packages/ai/package.json

```diff
@@ -21,24 +21,25 @@
     "directory": "packages/ai"
   },
   "scripts": {
-    "lint": "yarn lint-scripts",
+    "lint": "yarn lint-misc && yarn lint-scripts",
+    "lint-misc": "prettylint --ignore-path .gitignore '**/*.{json,md,toml}'",
```

Comment:
Open to script name suggestions. 🤔 I'm not sure what we could call it aside from misc since each file extension almost serves a different purpose, but I don't really like misc.

Reply:
Again, name is fine by me. Either this or `lint-files-that-do-not-end-in-py`. 🤷‍♂️ I'll let you take your pick of the two.

Reply:
If those are my only two options, then yeah `misc` is the better choice! 😂